<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimplifyTab</class>
 <widget class="QWidget" name="SimplifyTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>588</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="TextEdit" name="SimpExp">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Expression</string>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Expression</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SimpPrev">
         <property name="toolTip">
          <string>Preview the expression</string>
         </property>
         <property name="text">
          <string>Preview</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SimpCalc">
         <property name="toolTip">
          <string>Simplify the expression - Shift + Enter</string>
         </property>
         <property name="text">
          <string>Simplify</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTextBrowser" name="SimpOut">
      <property name="toolTip">
       <string>Simplified Answer</string>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::NoWrap</enum>
      </property>
      <property name="placeholderText">
       <string>Output</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>caspy3/qt_assets/highlight.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
